### PR TITLE
test: improve utils coverage

### PR DESCRIPTION
**Pull request by jupyterjazz**
closes https://github.com/jina-ai/now.private/issues/328

- removed unused functions
- moved some functions to more appropriate places
- wrote some unit tests

---

- [x] This PR references an open issue
- [x] Added a test
- [ ] Updated the documentation
- [ ] Added a screenshot of a successful customer deployment

### CODE REVIEW

File: now/run_all_k8s.py [hunk 1]
```diff
@@ -220,3 +219,31 @@
         results_per_table=results_per_table,
         disable_to_datauri=disable_to_datauri,
     )
+
+
+def get_flow_status(action, **kwargs):
```
Comment (joschkabraun) [inline now/run_all_k8s.py:224]: let's give this function a leading underscore to indicate it's private to the file

File: now/utils.py [hunk 2]
```diff
@@ -242,7 +150,7 @@ def get_aws_profile():
 
 
 def hide_string_chars(s):
-    return ''.join(['*' for _ in range(len(s) - 5)]) + s[len(s) - 4 :] if s else None
+    return ''.join(['*' for _ in range(len(s) - 4)]) + s[len(s) - 4 :] if s else None
```
Comment (joschkabraun) [inline now/utils.py:153]: why the change from `5` to `4`?

Comment (jupyterjazz) [reply now/utils.py:153]: 5 is incorrect, loses one character

**Review by IyadhKhalfallah — APPROVED** [review]
